src/api/Alert/index.ts

```diff
@@ -195,9 +208,62 @@ export class AlertAPI extends Request {
   }
 
   /**
-   * Do an action (open, dismiss, confirm, unconfirm) on a match.
+   * Update a comment on a match if the user that triggered the update is the author
```

Comment:
we should add a dot at the end of line for consistency with the other descriptions